compiler-core/templates/documentation_layout.html

```diff
@@ -1,18 +1,41 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" data-theme="light" data-drawer="closed" data-prewrap="off">
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <title>{{ page_title }}</title>
+    <meta name="description" content="{{project_name}} v{{project_version}} · {{page_meta_description}}"/>
+    <!-- Preload most used font variants -->
+    <link rel="font" as="font" href="{{ unnest }}/fonts/karla-v23-regular-latin.woff2" type="font/woff2"/>
+    <link rel="font" as="font" href="{{ unnest }}/fonts/karla-v23-bold-latin.woff2" type="font/woff2"/>
+    <link rel="font" as="font" href="{{ unnest }}/fonts/ubuntu-mono-v15-regular-latin.woff2" type="font/woff2"/>
+    <!-- Preload base script and version switcher -->
+    <link rel="script" as="script" href="{{ unnest }}/js/index.js?v={{ rendering_timestamp }}" type="application/javascript"/>
+    <link rel="script" as="script" href="{{ unnest }}/docs_config.js" type="application/javascript"/>
+    <!-- Preload source code highlighting -->
```

Comment:
I don't think we should load these first, we prioritise content over syntax highlighting

Reply:
Would be good if we can move in in all the things that stop cumulative layout shifts via HLJS application. Then the hljs application just adds colors and nothing else changes. I will remove this.